### ISSUE TITLE
consider semver for tagging

### ISSUE THREAD

**Issue by dvzrv** [ssj71/rkrlv2#35]
Hey there!

For packaging it would be nice to have proper tagged versions in semver (e.g. `0.1.0`), as the current tags are very non-standard.

Could you do a new release any time soon? Thanks! :)

**Comment by ssj71** [comment]
I'm much less confident in this project than my others. In fact I never actually did a "real" release because most of the effects are barely tested. When nobody really stepped up to help with testing each effect I lost steam on it and moved to other projects (though I still indend to someday polish each of these up). I know of several changes that need to be made that will affect the sound so I'm reluctant to do a full release. I suppose technically that semver would say its not actually released until it gets to 1.0.0 but that's going to be a while still.

Once I get to a release I'd be happy to use semver but I'll have to dig back into the dev branch to remember what the actual status of everything is and what should be finished before a release.

**Comment by trebmuh** [comment]
FWIW, I'm using "0.0.0+20171213~beta2" as a number for my package.
